kratos/utilities/auxiliar_model_part_utilities.h

```diff
@@ -350,11 +350,11 @@ class KRATOS_API(KRATOS_CORE) AuxiliarModelPartUtilities
     }
 
     /// To Export a Vector data (std::vector/array/..)
-    template<class TDataType>
+    template<class TContainerType, class TVarType>
```

Comment:
Curious: Wouldn't it be better to deduce `TVarType` from `TContainerType::value_type` as in the scalar case?

Reply:
because I need to turn Vector variables (such as array_3) into a vector